README.md

```diff
@@ -3,6 +3,18 @@
 This repository contains templates used to create refernce documentation for developers.google.com,
 cloud.google.com, and other sites.
 
+## Developing
+
+### Testing
+
+You can run the tests with the following command:
+
+```
+TRAMPOLINE_BUILD_FILE=./ci/run_tests.sh TRAMPOLINE_IMAGE=gcr.io/cloud-devrel-kokoro-resources/docfx ci/trampoline_v2.sh
```

Comment:
That doesn't actually look like a way to run tests.

Reply:
It runs the tests using Trampoline V2 (inside a Docker container). The `TRAMPOLINE_BUILD_FILE` variable says which script to run.

I updated the README to explain this and describe how to run the tests without Docker.